src/components/Conversations/ConversationsMessage.tsx

```diff
@@ -0,0 +1,53 @@
+import { createStyles, Group, Text, Box } from '@mantine/core';
+import { UserAvatar } from '~/components/UserAvatar/UserAvatar';
+import { useCurrentUser } from '~/hooks/useCurrentUser';
+import { UserWithCosmetics } from '~/server/selectors/user.selector';
+
+type ConversationsMessageProps = {
+  text: string;
+  sender: Partial<UserWithCosmetics>;
+};
+
+export function ConversationsMessage(props: ConversationsMessageProps) {
```

Comment:
Nit picking, but generally speaking, unless you're passing down the props to another component in it's entirety, we tend to spread the items and use them. i.e: `{ text, sender } : ConversationsMessageProps`